node/src/effect/requests.rs

```diff
@@ -360,36 +360,39 @@ impl Display for StateStoreRequest {
 
 /// Details of a request for a list of deploys to propose in a new block.
 #[derive(DataSize, Debug)]
-pub struct ListForInclusionRequest {
+pub struct ProtoBlockRequest {
     /// The instant for which the deploy is requested.
     pub(crate) current_instant: Timestamp,
     /// Set of deploy hashes of deploys that should be excluded in addition to the finalized ones.
-    pub(crate) past_deploys: HashSet<DeployHash>,
+    pub(crate) past_deploys: Vec<DeployHash>,
     /// The height of the next block to be finalized at the point the request was made.
     /// This is _only_ a way of expressing how many blocks have been finalized at the moment the
     /// request was made. Block Proposer uses this in order to determine if there might be any
     /// deploys that are neither in `past_deploys`, nor among the finalized deploys it knows of.
     pub(crate) next_finalized: u64,
+    // TODO
```

Comment:
Can you add more context to this `TODO`?

Reply:
Sorry, missed this - now a doc comment.